subsystem/identity_provider_resource_test.go

```diff
@@ -131,6 +131,55 @@ var _ = Describe("Identity provider creation", func() {
 		Expect(terraform.Apply()).To(BeZero())
 	})
 
+	It("Can create a 'github' identity provider", func() {
```

Comment:
please add tests that check 
* assignment for organizations and teams
* teams without the convention